<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoView</class>
 <widget class="QDialog" name="ProtoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pbText">
     <property name="text">
      <string>Текстовый</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pbBinary">
     <property name="text">
      <string>Бинарный</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pbHex">
     <property name="text">
      <string>Двоичный</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="chbEditable">
     <property name="text">
      <string>Редактирование</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextBrowser" name="textField"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
